CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Master
 
+## Bug Fixes
```

Comment:
There is already a `Bug Fixes` section below for this release.

Also, since this is fixing a bug there was introduced in master I don't think it makes sense to put it in the CHANGELOG. It wasn't present in the previous release either.

Reply:
So you propose to remove CHANGELOG record? Do I understand correctly?

Reply:
Yes, I don't think it makes sense to have a CHANGELOG entry for something that isn't a change since the last release.

Reply:
rollback finished && push forced